manifests/repo/community.pp

```diff
@@ -25,6 +25,17 @@
       sslcacert       => '/etc/pki/tls/certs/ca-bundle.crt',
       sslverify       => 1,
     }
+    if versioncmp($repo_release, '8') >= 0 {
```

Comment:
EL9 isn't coming out anytime soon :)

This would read easier by doing `if $repo_release == '8'` instead of `versioncmp()`

Reply:
Updated to just be equality check